projects/vaults/contracts/vaults/Symbiotic/InceptionVault_S.sol

```diff
@@ -219,6 +249,20 @@ contract InceptionVault_S is MellowHandler, IInceptionVault_S {
 
         emit Withdraw(claimer, receiver, claimer, amount, iShares);
     }
+        
+    function withdraw(uint256 assets, address receiver, address owner) public virtual whenNotPaused nonReentrant returns (uint256) {
```

Comment:
virtual?

Reply:
Incase we wanna inherit from InceptionVault_S in another contract ?

Reply:
I can't imagine such cases in the near feature, tbh.